### PR TITLE
Update 2000-01-01-run-scheduled-tasks.md

### PR DESCRIPTION
**Pull request by yannski**

### CODE REVIEW

File: _posts/platform/app/2000-01-01-run-scheduled-tasks.md [hunk 1]
```diff
@@ -1,28 +1,34 @@
...
 
-Your application may need to execute some tasks at regular interval. The cron feature is here to help you run scheduled tasks based on [one-off containers]({% post_url platform/app/2000-01-01-tasks %}).
+Your application may need to execute some tasks at regular interval. The cron feature is here to help you run scheduled
+tasks based on [one-off containers]({% post_url platform/app/2000-01-01-tasks %}).
 
-Instead of running 24/7 a container to execute a task from time-to-time, the cron feature lets you start one-off containers only when you need.
+Instead of running 24/7 a container to execute a task from time-to-time, the cron feature lets you start one-off
+containers only when you need.
+
+TBD how is it enabled
```
Comment (TheoEb) [inline _posts/platform/app/2000-01-01-run-scheduled-tasks.md:17]: No idea what to write here

File: _posts/platform/app/2000-01-01-run-scheduled-tasks.md [hunk 2]
```diff
@@ -1,28 +1,34 @@
...
-For instance, here is a file example of how to schedule a task every 10 minutes to execute the command `rails orders:check` on a 2XL container:
+For instance, here is a file example of how to schedule a task every 15 minutes to execute the command
+`rails orders:check` on a 2XL container:
 
 ```json
 {
   "jobs": [
     {
-      "command": "*/10 * * * * rails orders:check",
+      "command": "*/15 * * * * rails orders:check",
```
Comment (TheoEb) [inline _posts/platform/app/2000-01-01-run-scheduled-tasks.md:31]: I think we put a min interval at 15min, so I updated the example

File: _posts/platform/app/2000-01-01-run-scheduled-tasks.md [hunk 3]
```diff
@@ -47,3 +58,35 @@ scalingo --app my-app cron-tasks
...
+### How To Enable Cron Tasks?
+
+The cron feature is enabled by default.
+
+### How To Disable Cron Tasks?
+
+You may want to disable automatic cron tasks execution, for example for [review apps]({% post_url
+platform/app/2000-01-01-review-apps %}) or staging environments.
+
+TBD
```
Comment (TheoEb) [inline _posts/platform/app/2000-01-01-run-scheduled-tasks.md:80]: No idea of how to disable

File: _posts/platform/app/2000-01-01-run-scheduled-tasks.md (hunk 1 above)
Comment (EtienneM) [reply _posts/platform/app/2000-01-01-run-scheduled-tasks.md:17]: I think Yann meant "how do you enable the cron product", wondering whether there are some extra step to enable the cron.json detection or not. I think you can remove it:
```suggestion
```

File: _posts/platform/app/2000-01-01-run-scheduled-tasks.md [hunk 4]
```diff
@@ -47,3 +58,35 @@ scalingo --app my-app cron-tasks
 | */10 * * * * echo biniou |      |
 +--------------------------+------+
 ```
+
+## Delete Configured Cron Tasks
+
+In order to delete a cron tasks, you must delete the associated entry in the `cron.json`.
+If there is no more tasks, or if you want to remove all tasks, you can also remove the file.
```
Comment (EtienneM) [inline _posts/platform/app/2000-01-01-run-scheduled-tasks.md:65]:
```suggestion
In order to delete a cron task, you must delete the associated entry in the `cron.json`.
If there is no more task, or if you want to remove all tasks, you can also remove the file.
```

File: _posts/platform/app/2000-01-01-run-scheduled-tasks.md [hunk 5]
```diff
@@ -47,3 +58,35 @@ scalingo --app my-app cron-tasks
...
+
+The cron tasks configuration will be updated when redeploying the application.
+
+## FAQ
+
+### How To Enable Cron Tasks?
+
+The cron feature is enabled by default.
```
Comment (EtienneM) [inline _posts/platform/app/2000-01-01-run-scheduled-tasks.md:73]: Nope. You need to ask the support. It is not globally available yet. You should say that it is currently in beta state but it is possible to get access to it by asking the support.

File: _posts/platform/app/2000-01-01-run-scheduled-tasks.md (hunk 3 above)
Comment (EtienneM) [reply _posts/platform/app/2000-01-01-run-scheduled-tasks.md:80]: No idea either, can you ask Yann about this

File: _posts/platform/app/2000-01-01-run-scheduled-tasks.md [hunk 6]
```diff
@@ -47,3 +58,35 @@ scalingo --app my-app cron-tasks
...
+
+You may want to disable automatic cron tasks execution, for example for [review apps]({% post_url
+platform/app/2000-01-01-review-apps %}) or staging environments.
+
+TBD
+
+### How Precise Is The Cron Feature?
+
+Since the cron feature needs to start a new container to execute the task, the precision will depend on the
+deployment time.
```
Comment (EtienneM) [inline _posts/platform/app/2000-01-01-run-scheduled-tasks.md:85]: This is not true, it does not depend on the deployment time. We don't trigger a new deployment every time we execute a task. It depends on the image size as we need to pull the application image which may take a few minutes

File: _posts/platform/app/2000-01-01-run-scheduled-tasks.md [hunk 7]
```diff
@@ -47,3 +58,35 @@ scalingo --app my-app cron-tasks
...
+
+### How Will I Be Billed?
+
+The cron feature in itself is free. The one-off containers launched to run the commands defined in your cron tasks will
+be like other [one-off containers]({% post_url platform/app/2000-01-01-tasks %}) ie you will be billed for the type of
+container you defined in you cron task (M is the default container size) for the time it was executed. For example of
```
Comment (EtienneM) [inline _posts/platform/app/2000-01-01-run-scheduled-tasks.md:91]: This sentence is too long and should be split in shorter sentences.

"ie" is spelled "i.e."

We shouldn't use past tense in documentation

**Review by EtienneM — CHANGES_REQUESTED** [review]

File: _posts/platform/app/2000-01-01-run-scheduled-tasks.md [hunk 8]
```diff
@@ -33,9 +37,14 @@ For instance, here is a file example of how to schedule a task every 10 minutes
...
-  - The command is any command you can execute in a one-off container (i.e. with the command `scalingo --app my-app run <command>`).
-- `size`: specify the [container size]({% post_url platform/internals/2000-01-01-container-sizes %}) of the one-off container executing the command. This key is optional, it defaults to M (512 MiB RAM).
+  - The cron expression follows the [crontab standard](https://en.wikipedia.org/wiki/Cron#CRON_expression). You may 
+  find the website [crontab.guru](https://crontab.guru/#*/10_*_*_*_*) useful to write your own cron expression.
+  - The command is any command you can execute in a one-off container
+  (i.e. with the command `scalingo --app my-app run <command>`).
+- `size`: specify the [container size]({% post_url platform/internals/2000-01-01-container-sizes %}) of the one-off 
+container executing the command. This key is optional, it defaults to M (512 MiB RAM).
+
+Your cron expression must be set with at least 10 minutes interval.
```
Comment (yannski) [inline _posts/platform/app/2000-01-01-run-scheduled-tasks.md:47]:
```suggestion
**Your cron expression must be set with at least 10 minutes interval.**
```

File: _posts/platform/app/2000-01-01-run-scheduled-tasks.md [hunk 9]
```diff
@@ -1,22 +1,42 @@
...
 
-Instead of running 24/7 a container to execute a task from time-to-time, the cron feature lets you start one-off containers only when you need.
+The syntax used to describe this interval is the same as in the cron software.
 
-## Configuring Cron Tasks on Your Application
+Scalingo Scheduler launches tasks as [one-off containers]({% post_url platform/app/2000-01-01-tasks %}) in detached mode. Therefore the related one-off documentation and their detached mode applies.
 
-Configure cron tasks on your application by adding a `cron.json` file at the root of your application. The file will be detected at the next deployment and the cron task will be configured for your application.
+{% warning %}
+Scheduled tasks execution is expected but not guaranteed. Scalingo Scheduler is known to occasionally (but rarely) miss the execution of scheduled jobs. If scheduled tasks are a critical component of your application, it is recommended to [run a custom clock process]() instead for more reliability, control, and visibility.
```
Comment (yannski) [inline _posts/platform/app/2000-01-01-run-scheduled-tasks.md:20]: @EtienneM can you rephrase with your own words?
